miv/statistics/burst.py

```diff
@@ -0,0 +1,84 @@
+# Bursting_Analysis: Burst length, location, and rate
+
+##Bursting is defined as the occurence of a specified number of spikes (usually >10), with a small interspike interval (usually < 100ms)
+
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from miv.typing import SpikestampsType
+
+
+def burst(spiketrains: SpikestampsType, channel: float, min_isi: float, min_len: float):
+
+    # Calculates parameters critical to characterize bursting phenomenon on a singl channel
```

Comment:
Please use doctring format instead of comment.

Reply:
commit 79d453b277ebc59c276b7b0a3cfc2c564eac3715